variables.tf

```
@@ -68,6 +68,18 @@ variable "redis_timeout" {
   default     = "10m"
 }
 
+variable "use_redis_replicas" {
+  description = "Whether to enable redis replicas or not. Read replica mode. Can only be specified when trying to create the instance. If set to true, memory_size_gb variable value should be 5 or above"
+  type        = bool
+  default     = false
+}
+
+variable "redis_replica_count" {
```

Using a common prefix helps to group similar/related attributes together in the terraform module docs

```suggestion
variable "read_replicas_count" {
```